### DISCUSSION TITLE
Group Scenes: Struggling to understand Bottom Navigation with Precompose

### DISCUSSION THREAD

**Discussion by neorix11** [Tlaster/PreCompose#170] [General]
Hello, all. Looking to understand more about how `group` could potentially satisfy bottom navigation.  Does anyone have an example of this?

**Comment by Tlaster** [comment]
`group` is similar to [`navigation`](https://developer.android.com/jetpack/compose/navigation#nested-nav) in jetpack, which is not for bottom navigation, you can use a `NavHost` with your bottom navigation.

**Reply by neorix11** [reply]
Awesome.  Thanks!
